desktop_gateway_sg_rules.tf

```diff
@@ -0,0 +1,94 @@
+# Allow ingress from COOL Shared Services via ssh
+# For: DevOps ssh access to Guacamole instance
+resource "aws_security_group_rule" "desktop_gw_ingress_from_cool_via_ssh" {
+  provider = aws.provisionassessment
+
+  security_group_id = aws_security_group.desktop_gateway.id
+  type              = "ingress"
+  protocol          = "tcp"
+  cidr_blocks       = [local.cool_shared_services_cidr_block]
```

Comment:
See https://github.com/cisagov/cool-assessment-terraform/pull/5/commits/04a8e1f26a2552aed3dbd32fa31549a9b2320c06.

Reply:
Are you no longer using the OpenVPN instance as a jump box?  If not then your changes are correct.  If so then you need to add `<openvpn_server_ip>/32`.

This reminds me that I have to get OpenVPN + PIV working in Linux as well.

Reply:
Yup- I was thinking that the OpenVPN server IP was included in the `client_vpn_cidr_block`.  I will change this to `vpn_server_cidr_block` instead.  As @felddy pointed out in a chat, this is preferred over specifying only the IP address of the OpenVPN server because that IP will change whenever the instance is redeployed.  We don't want to have to reapply this terraform in every assessment environment when that happens.

This solution also covers us if multiple VPN servers are deployed for failover situations (as long as they are all within `vpn_server_cidr_block`).

Reply:
Take a look at https://github.com/cisagov/cool-assessment-terraform/pull/5/commits/dcd6a4c16ded570a9dea4da64dc55ab038e230a8 and let me know if this looks better.